lib/streamlit/DeltaGenerator.py

```diff
@@ -325,7 +308,7 @@ def __getattr__(self, name):
 
         def wrapper(*args, **kwargs):
             if name in streamlit_methods:
-                if self._container == BlockPath_pb2.BlockPath.SIDEBAR:
+                if self._container == "sidebar":
```

Comment:
why don't we use the proto value anymore?

Reply:
I have some reasons, but actually I'll just roll this back from the PR since it's not related to the issue the PR addresses.